SmileAuth/Classes/SmileAuthenticator.m

```diff
@@ -304,4 +304,7 @@ +(void)clearPassword{
     [[SmileAuthenticator sharedInstance].keychainWrapper writeToKeychain];
 }
 
+-(void)changeAuthentication:(BOOL)newAuth {
+    _isAuthenticated = NO;
+}
```

Comment:
why this method need a new parameter `(BOOL)newAuth`, but don't use it in scope?